sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -44,7 +44,7 @@ az cognitiveservices account create \
     --yes
 ```
 
-Interaction with this service begins with an instance of a [client](#client "ta-client").
+Interaction with this service begins with an instance of a [client](#TextAnalyticsClient "TextAnalyticsClient").
```

Comment:
can you check that this hook works with the readme generated on the github.io page? I recall that you can't use the same name

Reply:
thank you, fixed and can see they render now